RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -103,7 +97,10 @@ metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
 
        //dP4 recovery
        for( const auto& it : footprint) {
```

Comment:
```suggestion
       for( const auto& it : footprint) {
         // Special treatment for PUPPI with dR, since jet candidates (puppi) and MET candidates (puppiForMet)
         // can't be matched through the sourceCandidatePtrs and may have different energy, but same direction.
```
please keep the comments from the 106X version